MTL-1864 (#2163)
Cray-HPE/docs-csm: operations/configuration_management/Pre_Boot_NCN_Image_Configuration.md

1. [Create a CFS Configuration](Create_a_CFS_Configuration.md)

>    **NOTE:** If the platform certificate is needed for the purpose of accessing zypper repos,

Reviewer: ```suggestion
   **NOTE:** If the platform certificate is needed for the purpose of accessing Zypper repos,
```

>    the `csm.ncn.ca_cert` role can be added to your playbook within the csm-config-management repo.

Reviewer: ```suggestion
   the `csm.ncn.ca_cert` role can be added to a playbook within the `csm-config-management` repo.
```

>    ```console

Reviewer 2: This looks like it should be `yaml`.

    # Install the platform certificate
    - role: csm.ncn.ca_cert
   ```

>    The first layer in the CFS session should be similar to this, where <example-playbook.yml> is the playbook

Reviewer: ```suggestion
   The first layer in the CFS session should be similar to this, where `<example-playbook.yml>` is the playbook
```

>    that includes the csm.ncn.ca_cert role.

Reviewer: ```suggestion
   that includes the `csm.ncn.ca_cert` role.
```

>    ```console

Reviewer 2: This should be `json`.

   "layers": [
   {
     "name": "csm-config",
     "cloneUrl": "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git",
     "playbook": "<example-playbook.yml>",
     "commit": "<git commit id>"
   },
   ```

>    **NOTE:** There are three existing playbooks availabe for NCN'S

Reviewer: ```suggestion
   **NOTE:** There are three existing playbooks available for NCNs:
```

>    - ansible/ncn-worker_nodes.yml

Reviewer: ```suggestion
   - `ansible/ncn-worker_nodes.yml`
```

>    - ansible/ncn-storage_nodes.yml

Reviewer: ```suggestion
   - `ansible/ncn-storage_nodes.yml`
```

>    - ansible/ncn-master_nodes.yml

Reviewer: ```suggestion
   - `ansible/ncn-master_nodes.yml`
```
